modules/wri_paragraph/config/install/language.content_settings.paragraph.superfeatured_item.yml

```diff
@@ -0,0 +1,17 @@
+langcode: id
```

Comment:
The following paragraphs end up not having any translatable fields once we discard blocks and paragraph references, so you can skip them here too:
```
	config/language.content_settings.paragraph.image_cta.yml
	config/language.content_settings.paragraph.impact_in_action_blocks.yml
	config/language.content_settings.paragraph.listing_block.yml
	config/language.content_settings.paragraph.narrative_taxonomy.yml
	config/language.content_settings.paragraph.person_listing.yml
	config/language.content_settings.paragraph.small_cta.yml
	config/language.content_settings.paragraph.superfeatured_item.yml
	config/language.content_settings.paragraph.unique_feature_cta.yml
```